scripts/rno-g-simple-copy

```diff
@@ -53,7 +53,8 @@ do
   FREESPACE=`df /data | awk 'NR>1 {print $4}'`
   if [ $FREESPACE -lt $THRESH1 ] ;
   then
-    find /data/daq -${CMP} +${AGE1} -delete
+    # Try to copy a last time and delete if / after copied
+    find /data/daq -${CMP} +${AGE1} | rsync  -avh --remove-source-files --files-from=- /data/daq 10.1.0.1:/data/ingress/station$STN/
```

Comment:
have you checked that this rsync command does what we want? I'm not 100% sure how --files-from treats relative paths

Reply:
May be worth trying a test command with --dry-run if you haven't already

Reply:
okay I had to slightly change the command to
```
find /data/daq -mtime +5 | rsync  -navh --remove-source-files --files-from=-  / 10.1.0.1:/data/ingress/station$STN/
```

Last few lines:
```
data/daq/run240315/header/000067.hd.dat.gz
data/daq/run240315/header/000142.hd.dat.gz
data/daq/run240315/waveforms/
data/daq/run240315/waveforms/000000.wf.dat.gz
data/daq/run240315/waveforms/000067.wf.dat.gz
data/daq/run240315/waveforms/000142.wf.dat.gz

sent 42.01K bytes  received 3.91K bytes  10.20K bytes/sec
total size is 1.47G  speedup is 31,952.13 (DRY RUN)
```